api.bs

```diff
@@ -981,23 +985,37 @@ A <dfn>privacy budget key</dfn> is a [=tuple=] consisting of the following items
 
 To <dfn>deduct privacy budget</dfn>
 given a [=privacy budget key=] |key|,
-[=float=] |epsilon|,
+[[WEBIDL#idl-double|double]] |epsilon|,
 integer |sensitivity|,
 and integer |globalSensitivity|:
 
 1.  If the [=privacy budget store=] does not [=map/contain=] |key|, [=map/set=]
-    its value of |key| to be a [=user agent=]-defined value.
+    its value of |key| to be a [=user agent=]-defined value,
+    plus 1000.
+
+    <p class=note>The addition of 1000 to this value
+    ensures that the rounding errors added by this algorithm do not cause
+    the budget to be exceeded unnecessarily after multiple invocations.
+    The privacy loss from the additional one-thousandth of an epsilon is trivial.
 
 1.  Let |currentValue| be the result of [=map/get|getting the value=] of |key|
     in the [=privacy budget store=].
 
-1.  If |currentValue| is less than or equal to 0, return false.
+1.  Let |deductionFp| be |epsilon| * |sensitivity| / |globalSensitivity|.
+
+1.  If |deductionFp| is negative or greater than 4294,
```

Comment:
A note would be helpful to explain where `4294` is coming from. And since that value is also used below, maybe it should be pulled into a constant definition.